nuclio/export.py

```diff
@@ -41,6 +41,7 @@
 archive_settings = {}
 
 is_comment = re.compile(r'[ \t]*#.*').match
+is_annotation = re.compile(r'[ \t]*#[ \t]*(nuclio|mlrun):[ \t]*(return|ignore|start-code|end-code).*').match
```

Comment:
let's move `r'[ \t]*#[ \t]*(nuclio|mlrun):[ \t]*` to some variable instead of duplicating it